lib/commands/slicemachine/commands/init.js

```diff
@@ -201,8 +216,15 @@ async function init() {
       const [protocol, base] = ctx.base.split('://');
       if (!protocol || !base) return [`Base url "${ctx.base}" is invalid: did you forget to specify protocol?`, null];
 
-      const apiEndpointUrl = `${protocol}://${r.domain}.${ctx.base.replace(/^(?:https?:\/\/)?(?:www\.)?/i, '')}/api/v2`;
-      return [null, _format(r.domain, r.magicUrl, apiEndpointUrl)];
+      const apiEndpoint = `${protocol}://${r.domain}.${ctx.base.replace(/^(?:https?:\/\/)?(?:www\.)?/i, '')}/api/v2`;
+      const configFile = Mustache.render(
```

Comment:
can you do this part elsewhere? We are just computing the info that contains the apiEndpoint, you could write the file right after instead of mixing everything